systemd/ceph-osd@.service

```diff
@@ -19,6 +19,7 @@ TasksMax=infinity
 Restart=on-failure
 StartLimitInterval=30min
 StartLimitBurst=3
+RestartSec=5min
```

Comment:
In the scenario I have in mind, at boot time, ceph-disk@.service may happen 5 to 10 minutes after ceph-osd@.service attempted to run. There is no doubt in my mind that waiting minutes is useful. What I'm not 100% sure yet is if this can cause problem. Reason why I'm asking if you have a specific scenario in mind.